packages/web-frontend/src/containers/UserMenu/index.js

```diff
@@ -118,23 +148,62 @@ class UserMenu extends Component {
             horizontal: 'right',
           }}
         >
-          <MenuItem
-            onClick={() => {
-              openLandingPage();
-              this.closeUserMenu();
-            }}
-          >
-            View projects
-          </MenuItem>
+          {menuItems}
+        </Popover>
+      </div>
+    );
+
+    const ViewProjects = () => (
+      <MenuItem
+        onClick={() => {
+          openViewProjects();
+          this.closeUserMenu();
+        }}
+      >
+        View projects
+      </MenuItem>
+    );
```

Comment:
Dunno how standard it is to define inline components like this, but since they use internal state, I figure this is the best way to re-use components?

Reply:
Yeah if the component is small we can put it here.